curp/src/server/raw_curp/log.rs

```diff
@@ -165,18 +166,19 @@ impl<C: 'static + Command> Log<C> {
     }
 
     /// Get previous log entry's term and index
-    pub(super) fn get_prev_entry_info(&self, i: usize) -> (u64, usize) {
-        assert!(i > 0);
-        if i == 1 {
-            (0, 0) // fake log[0]
+    pub(super) fn get_prev_entry_info(&self, i: LogIndex) -> (LogIndex, u64) {
+        assert!(i > 0, "log[0] has no previous log");
+        if self.base_index == i - 1 {
+            (self.base_index, self.base_term)
         } else {
             let entry = self.get(i - 1).unwrap_or_else(|| {
                 unreachable!(
-                    "system corrupted, get log[{i}] when we only have {} log entries",
+                    "system corrupted, get log[{}] when we only have {} log entries",
+                    i - 1,
                     self.last_log_index()
                 )
             });
-            (entry.term, entry.index)
+            (entry.index, entry.term)
```

Comment:
Why reverse the order here, I see term in front of index in other places

Reply:
To unify with the format in other places